src/main/java/com/hr/app/jpaexample/controller/DepartmentController.java

```diff
@@ -32,5 +30,12 @@ public Department getDepartmentById(@PathVariable Long id) {
         return departmentService.getDepartmentById(id);
     }
 
+  // @GetMapping("/insert")
+   @PostMapping(value="/insert", consumes = "application/json", produces = "application/json")
```

Comment:
value="/insert" not required. Have you gone through the documentation I posted in group?

Reply:
I will change sir,